README.md

```diff
@@ -34,22 +34,41 @@ Apache header:
     WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
     See the License for the specific language governing permissions and
     limitations under the License.
-    
+
+
 ## Terraform Setup
 
-Terraform configurations are used to create the infrastructure needed for this integration, such as Pub/Sub topics and subscriptions, alerting policies, custom metrics, and Cloud Run instances. The `dev` and `prod` environments each have their own Terraform configurations, which can be found in the ```environments/``` directory. Terraform modules can be found in the ```modules/``` directory.
+Terraform configurations are used to create the infrastructure needed for this integration, such as Pub/Sub topics and subscriptions and Cloud Run instances. The `dev` and `prod` environments each have their own Terraform configurations, which can be found in the ```environments/``` directory. Terraform modules can be found in the ```modules/``` directory.
 
-Deployment with Terraform will be automated through source code changes in GitHub. To test the deployment manually, navigate to the desired environment folder (`environments/dev` or `environments/prod`) and run the following:
+Deployment with Terraform will be automated through source code changes in GitHub. To test the deployment manually, first create a Cloud Storage bucket in Cloud Shell that Terraform will use to store state:
```

Comment:
Is Cloud Storage used to store versioning information? I think that this is never explicitly state in the README. Maybe add a preceding sentence explaining this?

Reply:
Ah that's a good point, I've added it.